int64 grassNum
int64 eatAmount